care/emr/resources/questionnaire/utils.py

```diff
@@ -106,6 +106,32 @@ def validate_data(values, value_type, questionnaire_ref):  # noqa PLR0912
     return errors
 
 
+def normalize_value(value):  # noqa PLR0911
```

Comment:
Can you accept the type and parse based on the type, dynamic typic is kinda scary